lib/diaspora_federation/entities/comment.rb

```diff
@@ -21,6 +17,12 @@ class Comment < Entity
       #   Comment entity creation time
       #   @return [Time] creation time
       property :created_at, :timestamp, default: -> { Time.now.utc }
+
+      # Remove "created_at" when no order was received.
+      # @deprecated TODO: Remove this, this will break compatibility with pods older than 0.6.3.0.
```

Comment:
The can handle it, but they would generate the wrong formatted timestamp when relaying (That was fixed in 0.6.3.0), and that would break relaying for posts on old pods.